### ISSUE TITLE
amazon prime for subscription icon is broken

### ISSUE THREAD

**Issue by apedley** [HR10Knights/Should-I-Watch-This#135]
@apedley saw some episode info where the amazon prime icon for subscription was not showing. he cannot remember which show it was for.  i've added alt text to img tags to mitigate this, but something to look out for
